lib/plugins/aws/package/compile/events/httpApi/index.js

```diff
@@ -234,6 +236,7 @@ Object.defineProperties(
         routes,
         id: userConfig.id,
         metrics: userConfig.metrics || false,
+        throttle: Object.assign({ burstLimit: -1, rateLimit: -1 }, userConfig.throttle),
```

Comment:
Let's just assign config directly.

Also if property is not provided we should not define it in a template (unless we have really valid reason to do so - e.g. removing property doesn't clear the setting)

Reply:
Like this?